profitbricks/src/test/java/org/jclouds/profitbricks/features/DrivesApiLiveTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.profitbricks.features;
+
+import autovalue.shaded.com.google.common.common.collect.Iterables;
+import java.util.List;
+import org.jclouds.profitbricks.BaseProfitBricksLiveTest;
+import org.jclouds.profitbricks.domain.Drive;
+import org.jclouds.profitbricks.domain.Image;
+import org.jclouds.profitbricks.domain.Server;
+import static org.testng.Assert.assertNotNull;
+import org.testng.annotations.Test;
+
+@Test(groups = "live", testName = "DrivesApiLiveTest", singleThreaded = true)
+public class DrivesApiLiveTest extends BaseProfitBricksLiveTest {
+
+   public String serverId;
+   public String imageId;
+
+   @Override
+   protected void initialize() {
+      super.initialize();
+
+      List<Server> servers = api.serverApi().getAllServers();
+      assertNotNull(servers, "At least one server is required to run drives test.");
```

Comment:
Remove this assertion, as the server api never returns null. That's why it uses the fallback to an empty list. In jclouds we try to avoid returning null lists, so this is a pattern you can apply to the rest of the code.